src/model/services/mapi/legacy/credentials/key.ts

```diff
@@ -1,3 +1,11 @@
+import { decode } from 'js-base64';
+
 export const credentialsNamespace = 'giantswarm';
 
 export const defaultCredentialName = 'credential-default';
+
+export function decodeCredential(value?: string) {
+  if (!value) return undefined;
+
+  return decode(value);
```

Comment:
Using a 3rd party library because the `window`'s `btoa()` function does not support UTF-8 encoded strings :trollface: